backend/app.py

```diff
@@ -14,41 +14,35 @@
 CACHE_CONFIG = {"CACHE_TYPE": "filesystem", "CACHE_DIR": "/tmp"}
 cache = Cache(app, config=CACHE_CONFIG)
 
-AIRTABLE_API = "https://api.airtable.com/v0/"
-AIRTABLE_BASE = "appISQqfvRPMg6CH3/Suivi"
-API_KEY = os.getenv("AIRTABLE_API_KEY")
-HEADERS = {"Authorization": f"Bearer {API_KEY}"}
+NOTION_API = "https://api.notion.com/v1/databases/"
+NOTION_BASE = os.getenv("NOTION_BASE")
+API_KEY = os.getenv("NOTION_API_KEY")
+HEADERS = {
+    "Authorization": f"Bearer {API_KEY}",
+    "Content-Type": "application/json",
+    "Notion-Version": "2022-06-28"
+}
 CACHE_TIMEOUT = 1 if app.config['DEBUG'] else os.getenv('CACHE_TIMEOUT', 600)
 
 
 @app.route("/api/inventaire")
 @cache.cached(timeout=CACHE_TIMEOUT)
 def inventaire():
-    url = f"{AIRTABLE_API}{AIRTABLE_BASE}"
-    r = requests.get(url, headers=HEADERS)
-    return jsonify(r.json())
-
-
-@app.route("/api/demande", methods=["POST"])
-def demande():
-    # mapping {<form_name>: <airtables_name>}
-    fields = {
-        "description": "Données recherchées",
-        "title": "Titre du jeu de données",
-        "organisation": "Organisation",
-        "geo": "Echelle géographique souhaitée",
-        "update": "Fréquence de mise à jour souhaitée",
-        "reuse": "Réutilisation envisagée",
-        "contact": "Contact",
+    url = f"{NOTION_API}{NOTION_BASE}/query"
+    payload = {
```

Comment:
It could be nice if we could specify a set of properties to retrieve, so that custom columns such as `To do Mario` would not be returned by our API

Reply:
We can filter the properties and add the `key` here but we can't remove most of the mapping on vitepress side because it's used to format content in html. Also @agarrone will rework the Notion base so maybe we can use property with names that can be publicly seen instead of internal ones.

Reply:
Done